src/wp-includes/block-patterns.php

```diff
@@ -158,11 +158,33 @@ function _register_core_block_patterns_and_categories() {
 	);
 }
 
+/**
+ * Normalize the pattern from the API (snake_case) to the format expected by `register_block_pattern` (camelCase).
+ *
+ * @since 6.2.0
+ *
```

Comment:
```suggestion
 * @access private
 *
```

Marking as internal to Core-only private.